src/main/java/com/spotinst/sdkjava/model/api/azure/statefulNode/ApiStatefulNodeLaunchSpecification.java

```diff
@@ -0,0 +1,138 @@
+package com.spotinst.sdkjava.model.api.azure.statefulNode;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiStatefulNodeLaunchSpecification implements IPartialUpdateEntity {
```

Comment:
BootDiagnostics, Customdata, loadbalancerconfig,managedserviceidentities, secrets, shutdown script parameters missing

Reply:
Need clarification on this